src/containers/Stories/index.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Container } from "react-bootstrap";
+import SingleStoryCard from "../../components/SingleStoryCard";
+import LoadingPage from "../../components/LoadingPage";
+
+const Stories = () => {
+  const [posts, setPosts] = React.useState([]);
+  const _URL =
+    "https://public-api.wordpress.com/wp/v2/sites/nobarriers650349093.wordpress.com/posts";
+
+  React.useEffect(() => {
```

Comment:
We should decide to use React.hook or directly the hook during the bug fix week

Reply:
I don't think it's that problomatic, but yes we should